webxray/src/mix-master.js

```diff
@@ -204,7 +219,7 @@
                     // The dialog may have decided to replace all our spaces
                     // with non-breaking ones, so we'll undo that.
                     var html = data.endHTML.replace(/\u00a0/g, " ");
-                    var newContent = self.replaceElement(focusedElement, html);
+                    var newContent = self.replaceElement(originalElement, focusedElement, html, data.finished);
```

Comment:
why do we need to pass in originalElement when it's a direct reference ot focusedElement (rather than a cloned node)?